api/src/main/webapp/static/aou-file-tree-policy-extension.js

```diff
@@ -31,6 +34,35 @@ define([
         'data use policy by typing "affirm" below.');
       return !!affirm && 'affirm' === affirm.replace(/\W/g, '').toLowerCase();
     });
+
+    // Clicking the view button will
+    // 1) Open All Jupyter files in a new tab.
+    // 2) Display a download policy prompt for all other files as they will be downloaded
+    $('#aou-view-button-overlay').click(() => {
+      var isJupyterFile = true;
+      $('#notebook_list input:checked').each(function() {
+        // Find the selected file name
+        const elem = $(this).siblings().find($('.item_link .item_name'));
+        const fileName = elem.text();
+        var fileExt = fileName.substring( (fileName.lastIndexOf('.') +1) );
+        if (fileExt !== 'ipynb') {
+          isJupyterFile = false;
+          return false;
+        }
+       });
+      // If all selected Files are jupyter files return now
+      if (!!isJupyterFile) {
+        return true;
+      }
+
+      const affirm = prompt('The All of Us Data Use Policies prohibit you from removing participant-level data from ' +
```

Comment:
this works but I see that this duplicates the Download button behavior.  Can we make this common code?